recipes/sos-python/meta.yaml

```
@@ -0,0 +1,46 @@
+{% set name = "sos-python" %}
+{% set version = "0.18.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 360dd16db42664d3a609037354bd92c435a246510079621c906be9203338051c
+
+build:
+  noarch: python
+  number: 0
+  script:
+    - {{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - python >=3.6
+    - sos-notebook >=0.19.4
+    - sos >=0.19.8
+
+test:
+  requires:
+    - nose
+    - pytest
+    - selenium
+  imports:
+    - sos_python
+
+about:
+  home: https://github.com/vatlab/sos-python
+  license: BSD
+  license_family: BSD
+  license_file: LICENSE
+  summary: "SoS language module for Python2 and Python3"
```

It seems the package description on GitHub/PyPI is outdated:
```suggestion
  summary: "SoS language module for Python3"
```

---

This module does work for both Python2 and Python3. The PyPI description has a typo that I just fixed.